pallets/passkey/src/lib.rs

```diff
@@ -69,7 +73,7 @@ pub mod module {
 	}
 
 	#[pallet::event]
-	#[pallet::generate_deposit(fn deposit_event)]
+	// #[pallet::generate_deposit(fn deposit_event)]
```

Comment:
Question: We we want to add an event in the extrinsic for additional visibility?

Reply:
I'll add the event then